File: Framework/Qt/Python/Chapter_01/QT01_02.py
"""
Setup Qt application software main window.
"""

from PySide2.QtWidgets import QApplication, QMainWindow

if __name__ == '__main__':
    app = QApplication()

    # DECLARE Qt application software's main window.
    window = QMainWindow()

    '''
    INSTALL QLayout to place QWidgets!
    '''

    # PRESENT MainWindow and its child widgets of application software.
    window.show()

    app.exec_()

File: Framework/Qt/Cpp/Chapter_01/QT01_02/QT01_02.cpp
/*
Setup Qt application software main window.
*/

#include <qapplication.h>
#include <qmainwindow.h>

int main(int argc, char **argv) {
	QApplication app(argc, argv);

	// DECLARE Qt application software's main window.
	QMainWindow window;

	/*
	INSTALL QLayout to place QWidgets!
	*/

	// PRESENT MainWindow and its child widgets of application software.
	window.show();

	return app.exec();
}